lib/cypress-support/hooks.js

```diff
@@ -1,15 +1,10 @@
 /* eslint-disable no-param-reassign */
-let fetchPolyfill;
+const fetchPolyfill =
```

Comment:
Why not load this from the file, given that you've added it as an unused dependency? Or get rid of the dependency if inlining here

Reply:
> Why not load this from the file, given that you've added it as an unused dependency? Or get rid of the dependency if inlining here

Removed the dep